apps/emqx_authn/test/emqx_authn_mysql_SUITE.erl

```diff
@@ -258,20 +258,6 @@ raw_mysql_auth_config() ->
 
 user_seeds() ->
     [
-        #{
-            data => #{
-                username => "plain",
-                password_hash => "plainsalt",
-                salt => "salt",
-                is_superuser_str => "1"
-            },
-            credentials => #{
-                password => <<"plain">>
-            },
-            config_params => #{},
-            result => {error, not_authorized}
```

Comment:
Maybe those test cases are still valid for the fix as well?

Reply:
Yes, but IMO they do not add much if we do not raise a error